.travis.yml

```diff
@@ -8,9 +8,12 @@ before_script:
 
 go: tip
 script:
+  - test -z "$(gofmt -s -l $(find ./arm/* -type d -print) | tee /dev/stderr)"
   - test -z "$(gofmt -s -l -w management | tee /dev/stderr)"
   - test -z "$(gofmt -s -l -w storage | tee /dev/stderr)"
   - go build -v ./...
+  - test -z "$(go vet $(find ./arm/* -type d -print) | tee /dev/stderr)"
```

Comment:
Is there a shorter form of `go vet` that doesn't require you to do `$(find ...)`? Maybe like  `go vet -v ./arm` or `go vet -v ./arm/...` ?